src/components/Header.js

```diff
@@ -0,0 +1,40 @@
+import { Link, NavLink, useLocation } from "react-router-dom";
+import styles from "./Header.module.css";
+import pandaTypoImg from "assets/images/panda_typo.svg";
+import profileImg from "assets/images/ic_profile.svg";
+
+const Header = () => {
+  const location = useLocation();
+
+  const isActiveForItem =
+    location.pathname === "/items" || location.pathname === "/additem"
+      ? styles.active
+      : "";
+
+  return (
+    <header className={styles.header}>
+      <ul className={styles.menu}>
+        <li>
+          <Link to="/">
+            <img
+              src={pandaTypoImg}
+              className={styles.logo}
+              alt="판다마켓 로고"
+            />
+          </Link>
+        </li>
+        <li>자유게시판</li>
+        <li>
+          <NavLink to="/items" className={isActiveForItem}>
+            중고마켓
+          </NavLink>
```

Comment:
P1:
NavLink 태그에서는 아래와 같은 기능을 제공합니다.
관련 문서를 한번 보세요~


```suggestion
          <NavLink to="/items" className={({isActive}) => isActive ? styles.active : ''}>
            중고마켓
          </NavLink>
```

*React router dom NavLink
https://reactrouter.com/en/main/components/nav-link